hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -130,6 +132,7 @@
 			Pair<AccountID, TokenID>,
 			TokenRelProperty,
 			MerkleTokenRelStatus> tokenRelsLedger = null;
+	private TransactionalLedger<TokenID, TokenProperty, MerkleToken> tokensLedger = null;
```

Comment:
Afaict now we can (and probably should 😁 ) remove this field from `HederaLedger`.